package.json

```diff
@@ -25,7 +25,7 @@
         "@dicebear/core": "^9.2.2",
         "@headlessui/react": "^1.7.19",
         "@headlessui/tailwindcss": "^0.2.1",
-        "@hookform/resolvers": "^3.9.1",
```

Comment:
Q: why are we installing an older version?

Reply:
Was when trying to resolve the error with zod, but I guess can bump now